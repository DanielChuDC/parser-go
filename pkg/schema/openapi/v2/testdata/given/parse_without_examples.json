{
  "payload": {
    "example": {
      "test": "example1"
    },
    "properties": {
      "test": {
        "description": "hello",
        "example": "hello"
      }
    },
    "additionalProperties": {
      "description": "test",
      "example": "test"
    }
  }
}