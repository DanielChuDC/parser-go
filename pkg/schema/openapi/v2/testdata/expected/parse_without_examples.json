{
  "payload": {
    "additionalProperties": {
      "description": "test",
      "examples": [
        "test"
      ]
    },
    "properties": {
      "test": {
        "description": "hello",
        "examples": [
          "hello"
        ]
      }
    },
    "examples": [
      {
        "test": "example1"
      }
    ]
  }
}